std_msgs/Header header
geometry_msgs/Transform origin_transform
float64 cell_size
float32 OOB_occupancy_value
uint32 OOB_component_value
geometry_msgs/Vector3 dimensions
uint8[] data
